Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocDateTimeValueTest.cs

```diff
@@ -142,6 +142,17 @@ public void SetValueToNullableDateTimeNull ()
       Assert.That(_bocDateTimeValue.IsDirty, Is.True);
     }
 
+#if NET6_0_OR_GREATER
+    [Test]
+    public void IBusinessObjectBoundControl_SetValueToDateOnly_ThrowsArgumentException ()
+    {
+      DateOnly dateOnly = new DateOnly(2006, 1, 1);
+      _bocDateTimeValue.IsDirty = false;
+      Assert.That(() => { ((IBusinessObjectBoundControl)_bocDateTimeValue).Value = dateOnly;},
+          Throws.InstanceOf<ArgumentException>()
+              .With.Message.EqualTo("Parameter 'value' has type 'System.DateOnly' when type 'System.Nullable`1[System.DateTime]' was expected. (Parameter 'value')"));
```

Comment:
Please with to the dedicated ArgumentException assertion:
`Throws.ArgumentException.With.ArgumentExceptionMessageEqualTo(messsage, parametername)`